src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/PublicationController.cs

```diff
@@ -64,10 +64,10 @@ public async Task<ActionResult<List<PublicationSummaryViewModel>>> ListPublicati
 
         [HttpGet("api/publications/{publicationId}")]
         public async Task<ActionResult<PublicationViewModel>> GetPublication(
-            [Required] Guid publicationId)
+            [Required] Guid publicationId, [FromQuery] bool permissions = false)
```

Comment:
At this point in reading through the PR, it's unclear what the purpose of the "permissions" flag is here.

Presumably it'll decide whether or not to include a user's permissions with the returned object?  If so, I'd be tempted to rename this to `includePermissions` or `includeUserPermissions`.

Reply:
Disagree somewhat here. We're already using this flag for a bunch of other endpoints, so this would introduce inconsistency at this point. Not sure it's worth the effort to change this.

Personally, don't think it's particularly unclear what the purpose is and it's not super hard to follow it through to figure it out. This said, could be convinced that `includePermissions` is better, but `includeUserPermissions` seems like overkill.

Reply:
Discussed this on Slack. I've switched to using `includePermission` and changed it for all endpoints.